### DISCUSSION TITLE
[10일차 - 예진] undefined와 null의 차이

### DISCUSSION THREAD

**Discussion by ez1n** [zzangCS/zzangInterview#35] [프론트 면접 질문]
undefined와 null의 차이는 무엇인가요?

**Comment by skyhanull** [comment]
undefined은 변수를 선언하고 값을 할당하지 않은 상태, null은 변수를 선언하고 빈 값을 할당한 상태(빈 객체)이다. 즉, undefined는 자료형이 없는 상태입니다.

**Comment by ez1n** [comment]
**undefined**는 선언한 후에 값을 할당하지 않은 변수나 값이 주어지지 않은 인수에 할당되는 값으로 값이 결정되지 않았다는 뜻을 가진 값이다.

**null**은 어떤 값이 의도적으로 비어있음을 표현하는 값으로 해당 변수가 어떠한 객체도 가리키고 있지 않다는 것을 의미한다.
